skeleton/frontend/src/Frontend.hs

```diff
@@ -26,6 +29,7 @@ frontend = Frontend
   , _frontend_body = do
       el "h1" $ text "Welcome to Obelisk!"
       el "p" $ text $ T.pack commonStuff
+      prerender_ blank $ liftJSM $ void $ eval ("console.log('Hello, World!')" :: T.Text)
```

Comment:
Let's add a comment explaining that this is an *example* of using prerender specifically. And maybe alert really would be a better example since `print` and `putStrLn` can technically also be used for logging and we don't want to imply that this is *the* way to log.